<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArmyLayout</class>
 <widget class="QWidget" name="ArmyLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 15, 15);
font: 14pt &quot;Ancient Modern Tales&quot;;
color: rgb(230, 230, 230);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lbl_title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="UnitLayout" name="wgt_unit_1_1" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="UnitLayout" name="wgt_unit_1_2" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="UnitLayout" name="wgt_unit_2_1" native="true"/>
   </item>
   <item row="3" column="1">
    <widget class="UnitLayout" name="wgt_unit_2_2" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UnitLayout</class>
   <extends>QWidget</extends>
   <header>ui/unit_layout.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
